x-pack/plugins/actions/server/lib/task_runner_factory.ts

```diff
@@ -193,6 +194,14 @@ export class TaskRunnerFactory {
           }
         }
       },
+      cancel: async () => {
+        logger.warn(
+          `Action task with params '${stringify(
```

Comment:
I'm a little worried about writing out all the task instance params like this - I think today we're safe, for rules, since we seem to reference the id of the action task params saved object, but my worry is that some day someone will put something sensitive in the `params` and then we'd be exposing it in the logs.

And I think we'd want the task id and task type in the message as well.  Perhaps those are the only things we can safely add?

Reply:
Makes sense.